libgalois/src/analytics/pagerank/pagerank.cpp

```diff
@@ -58,23 +58,26 @@ katana::analytics::PagerankStatistics::Print(std::ostream& os) {
 katana::Result<katana::analytics::PagerankStatistics>
 katana::analytics::PagerankStatistics::Compute(
     katana::PropertyGraph* pg, const std::string& property_name) {
+  katana::GReduceMax<PRTy> max_rank;
+  katana::GReduceMin<PRTy> min_rank;
+  katana::GAccumulator<PRTy> distance_sum;
```

Comment:
Not a fan of C-style early/forward declaration.

Reply:
Yes, it was for another version. I will update this